Makefile

```diff
@@ -11,7 +11,7 @@ export GO111MODULE=on
 
 .PHONY: vendor vetcheck fmtcheck clean build gotest
 
-all: vendor vetcheck fmtcheck build gotest mod-clean
```

Comment:
It's better to skip a package instead of removing fmtcheck at all in case if it's hard to fix style and stuff

Reply:
I've tried all possible ways - skip, ignore, fix, upgrade. Nothing helps.